cmd/podman/pod_create.go

```diff
@@ -3,11 +3,15 @@ package main
 import (
 	"fmt"
 	"os"
+	"strconv"
 	"strings"
 
 	"github.com/containers/libpod/cmd/podman/libpodruntime"
 	"github.com/containers/libpod/cmd/podman/shared"
 	"github.com/containers/libpod/libpod"
+	"github.com/containers/libpod/pkg/rootless"
+	"github.com/cri-o/ocicni/pkg/ocicni"
```

Comment:
Late to the game. Is there a reason why `github.com/cri-o/ocicni/pkg/ocicni` is not part of `github.com/containers` (or ideally a package of libpod)? Would be nice for CI and for consolidating packages.